packages/babel-plugin-transform-typescript/src/index.js

```diff
@@ -190,10 +144,63 @@ export default declare((api, { jsxPragma = "React" }) => {
         // We do this here instead of in a `ClassProperty` visitor because the class transform
```

Comment:
Nit: this comment is slightly out of date, now that this block contains both `ClassProperty` and `ClassMethod` logic.